bigframes/session/__init__.py

```diff
@@ -407,6 +414,7 @@ def _read_gbq_query(
                         ],
                         "job_id": [query_job.job_id if query_job else "unknown"],
                         "location": [query_job.location if query_job else "unknown"],
+                        "use_query_cache": [use_cache],
```

Comment:
I don't think this is particually useful for DDM or DML queries. Let's remove.
```suggestion
```

More important is to pass `use_cache` to `_query_to_destination` and populate the QueryJobConfig.